src/xcode/ENA/ENA/Source/Scenes/ContactDiary/DiaryService.swift

```diff
@@ -6,43 +6,60 @@ import Foundation
 import Combine
 
 struct ContactPersonEncounter {
-	let id: Int
+	let id: Int64
 	let date: String
-	let contactPersonId: Int
+	let contactPersonId: Int64
 }
 
 struct LocationVisit {
-	let id: Int
+	let id: Int64
```

Comment:
Whats the reason for explicit Int64?

Reply:
FMDB is returning Int64.

Reply:
Can't we cast it then on read/write and just use plain Int (which is basically `Int64`).

Reply:
Since there is no 32Bit device in our range, I change it to Int.